packages/core/src/config/index.ts

```diff
@@ -5,4 +5,3 @@ export * from './builtIn.js';
 export * from './load.js';
 export * from './utils.js';
 export * from './config-resolvers.js';
-export * from './constants.js';
```

Comment:
Exporting it explicitly in the index file: https://github.com/Redocly/redocly-cli/pull/2308/files#diff-9a4ceebe7c6f86856371906c3f061d3b56b7457022b05179884a113e7ced67e8R42